### DISCUSSION TITLE
游戏运行是报错进行自动修复

### DISCUSSION THREAD

**Discussion by creeppppper** [Meloong-Git/PCL#623] [提问] [answered]
模组不兼容报错修复或缺少文件自动补全

**Comment by WForst-Breeze** [comment, marked as answer]
1. 模组兼容性检查功能请见 Discussions #619 
2. 我认为目前的高级管理中的手动补全文件已经足够玩家日常使用了，自动补全有点多此一举了
hmmm，但你迫切想要的话……也可以交个[ 优化建议 ](https://github.com/Hex-Dragon/PCL2/issues/new?assignees=&labels=%C2%B7+%E4%BC%98%E5%8C%96%2C%E6%96%B0%E5%8F%8D%E9%A6%88&template=ch.yml)啦qwq……
